tests/integration/security/crl/main_test.go

```diff
@@ -0,0 +1,114 @@
+//go:build integ
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crl
+
+import (
+	"fmt"
+	"testing"
+
+	"istio.io/istio/pkg/config/protocol"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/echo"
+	"istio.io/istio/pkg/test/framework/components/echo/common/deployment"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/tests/integration/security/crl/util"
+)
+
+var (
+	certBundle *util.Bundle
+	clientNS   namespace.Instance
+	serverNS   namespace.Instance
+	client     echo.Instance
+	server     echo.Instance
+	configs    []echo.Config
+	apps       deployment.TwoNamespaceView
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Nice test! does this really need to be a new test suite though?

Reply:
Yes. This involves revoking the certificate which causes errors for other tests for obvious reasons.